new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -257,6 +259,46 @@ public static V1DeploymentList listDeployments(String namespace) {
   }
 
   // --------------------------- pods -----------------------------------------
+  
+  /**
+   * Returns the V1Pod object given the following parameters.
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
```

Comment:
labelSelector in the format "weblogic.domainUID in (%s)", can be null

Reply:
yes, labelSelector can be a null String